util/util.go

```diff
@@ -88,3 +99,40 @@ func StringInSlice(value string, list []string) bool {
 	}
 	return false
 }
+
+//Int64InSlice returns true if the int64 is an element of the array
+func Int64InSlice(value int64, list []int64) bool {
```

Comment:
It seemed the `int64` linear searches did not take up too much time (mainly used when unique'ing unique connection timestamps). My guess is that the go compiler is smart enough to optimize the linear scan using SIMD or that the CPU is smart enough to speculatively execute checks in parallel.  (Moved from fsimporter)